en/docs/get-started/quick-start-guide.md

```diff
@@ -1,124 +1,134 @@
     | **GitHub Account**    | Your account                                  |
     | **GitHub Repository** | **`choreo-examples`** |
-    | **Branch**            | **`version-1`**                               |
-    | **Build Preset**      | Click **Ballerina** because you are creating the REST API from a [Ballerina](https://ballerina.io/) project and Choreo needs to run a Ballerina build to build it.|
+    | **Branch**            | **`main`**                               |
+    | **Build Preset**      | Click **Ballerina** because you are creating the REST API from a [Ballerina](https://ballerina.io/) project, and Choreo needs to run a Ballerina build to build it.|
     | **Path**              | **`reading-list-service`**                    |
 
-9. Click **Create** to initialize a REST API with the implementation from your GitHub repository.
-
-The REST API opens on a separate page where you can see its overview.
+8. Click **Create** to initialize the service with the implementation from your GitHub repository.
 
-###  Step 1.2: Deploy the REST API
+This takes you to a page with an overview of the service.
 
-For the REST API to be invokable, you need to deploy it. To deploy the REST API, follow the steps given below:
+###  Step 1.2: Deploy the service
 
-1. Navigate to the Choreo Console. You will be viewing an overview of the `readingList` REST API.
+For the REST endpoint of the service to be invokable, you need to deploy it. To deploy the service, follow the steps given below:
 
-2. In the left pane, click **Deploy**, and then click **Configure & Deploy**.
+1. In the left navigation menu, click **Deploy**.
 
-    !!! info
-        Automatic deployment is enabled for the REST API by default. You are required to carry out only the first deployment manually.<br/><br/> When automatic deployment is enabled, your REST API is automatically deployed every time you push a commit to the GitHub repository in which its implementation resides. You can disable automatic deployment if required.
+2. In the **Build Area** card, click **Deploy Manually**.
         
-3. In the **Configure & Deploy** pane, click **Deploy** without entering a sandbox endpoint. 
+3. In the **Configure & Deploy** pane that opens, you can see the **Readinglist** endpoint ready to be deployed. Click the edit icon next to the **Readinglist** endpoint.
+
+4. Change the **Network Visibility** to **Public**. This allows you to expose the endpoint to your web application securely.
+
+5. Click **Update**.
 
     !!! info
-        In this example, you are testing the REST API only in the Choreo Console and not in a sandbox environment. Therefore, you do not need to enter a sandbox endpoint.
+         In this example, you deploy a Ballerina service as a REST endpoint. Therefore, the REST endpoint is generated automatically. If you deploy a non-Ballerina service, you must manually add the REST endpoint and set the network visibility to **Public**.
```

Comment:
deployed?

Reply:
Shall we point them to the endpoint page, For e.g. learn more about endpoints or something along that lines?

Reply:
"Deployed" <- - - At this point, the service is not deployed. it will be deployed via the next step. 

+1 to point to the info on endpoints. Added a learn more link.